dev/benchmarks/openmp/example_standalone.py

```diff
@@ -65,8 +65,8 @@
 
 
 spike_mon = SpikeMonitor(P)
-state_mon = StateMonitor(S, 'w', record=range(10), when=Clock(dt=0.1*second))
-v_mon     = StateMonitor(P, 'v', record=range(10))
+state_mon = StateMonitor(S, 'w', record=list(range(10)), when=Clock(dt=0.1*second))
+v_mon     = StateMonitor(P, 'v', record=list(range(10)))
```

Comment:
This is a result of using `2to3` which is quite conservative and so wraps `range()` in `list()` to err on the side of caution (since in Python 2 it would have returned a list not a generator). I have overridden it in places where I felt it was safe for the sake of brevity, but left it in a few cases, such as this, where I wasn't sure. Should I change them all back or can you can spot any examples that should be wrapped like this?

Reply:
For the record argument it probably needs to be a list not an iterator (@mstimberg?) but ``arange`` is equivalent to ``array(range(10))`` so might be better here. For the ones below with map, it depends on whether or not matplotlib accepts iterators (haven't checked). My guess is they do so just the word list can be removed. Only way to check is to run the code though.